packages/peregrine/lib/talons/Gallery/useGalleryItem.js

```diff
@@ -0,0 +1,4 @@
+export const useGalleryItem = props => {
+    const handleLinkClick = () => {};
```

Comment:
I mean this is a little strange of course. I'm assuming your extension overrides this?

Reply:
Correct :)

Reply:
If we're going to attach a click handler we may as well attempt to invoke some sort of callback that may be passed, incase someone wanted to implement `GalleryItem` and pass an `onClick` handler.

```suggestion
    const { onClick = () => {} } = props;
    const handleLinkClick = useCallback((e) => {
        if (onClick) onClick(e);
    }, [onClick]);
```

Reply:
I'm also interested in what @jimbo has to say about this - we haven't yet implemented a function for the sake of having a target to wrap, so this would be the first.

Reply:
> Since an extension that wraps a talon may include new properties and not just override existing properties, and since undefined may be passed to a React element's onClick, we really don't even need to define a noop function as the default value. This talon could literally return an empty object.

I'd much prefer this over returning a no-op function here.